docs/assets/cgi/LICENSE.md

```diff
@@ -0,0 +1,2 @@
+Copyright (C) Andrew Savchenko - All Rights Reserved
+All files within this folder are proprietary and reserved for the use by TaskChampion project.
```

Comment:
It looks like these snuck into this PR, perhaps because it's based on `main`?

Reply:
Indeed. This belongs to https://github.com/taskchampion/taskchampion/pull/218